astropy/wcs/utils.py

```diff
@@ -72,6 +91,9 @@ def _wcs_to_celestial_frame_builtin(wcs):
     # Import astropy.time here otherwise setup.py fails before extensions are compiled
     from astropy.time import Time
 
+    solar_system_body_frames = {}
```

Comment:
We want these to be global variables, so that we define the types only once, so they should be outside of the function.